langchain/src/document_loaders/cheerio_web_base.ts

```diff
@@ -47,9 +47,9 @@ export class CheerioWebBaseLoader
     );
   }
 
-  async load(): Promise<Document[]> {
+  async load(selector?: SelectorType): Promise<Document[]> {
     const $ = await this.scrape();
-    const text = $("body").text();
+    const text = $(selector).text() ?? $("body").text();
```

Comment:
This should instead be `$(selector ?? "body").text();`

Reply:
Nice!